starknet_py/tests/e2e/client/gateway_test.py

```diff
@@ -74,6 +77,25 @@ async def test_estimate_fee_bulk(
         assert estimated_fee.overall_fee > 0
 
 
+@pytest.mark.asyncio
+async def test_get_compiled_class_by_class_hash(
+    gateway_client, hello_starknet_class_hash_tx_hash: Tuple[int, int]
+):
+    (class_hash, _) = hello_starknet_class_hash_tx_hash
+
+    compiled_class = await gateway_client.get_compiled_class_by_class_hash(
+        class_hash=class_hash
+    )
+
+    assert isinstance(compiled_class, CasmClass)
+    assert isinstance(compiled_class.prime, int)
+    assert isinstance(compiled_class.bytecode, list)
+    assert isinstance(compiled_class.hints, list)
+    assert isinstance(compiled_class.pythonic_hints, list)
+    assert isinstance(compiled_class.compiler_version, str)
+    assert isinstance(compiled_class.entry_points_by_type, CasmClassEntryPointsByType)
+
+
```

Comment:
I suppose this endpoint is not present in rpc?

Reply:
Exactly